tools/data_source/upload.xml

```diff
@@ -59,6 +59,7 @@
     </upload_dataset>
     <param name="force_composite" type="hidden" value="false" />
     <param name="dbkey" type="genomebuild" label="Genome" />
+    <param name="tags" type="hidden" value="" />
```

Comment:
Should this be in this PR?

Reply:
yes - it is required to write tests for group_tag parameters. This is in https://github.com/galaxyproject/galaxy/pull/18654 is isolation but which I've included in this PR as well.